.github/actions/spell-check/expect.txt

```diff
@@ -1879,6 +1890,7 @@ Secur
 securityoverview
 Segoe
 Sekan
+sektion
```

Comment:
This is also problematic since it's a spell error.

Reply:
these spell-checking errors must be happening because of a link i included in the code i may change it later. i will remove the spell-checking commit for now